### PR TITLE
Update customer BZ in docstring

### PR DESCRIPTION
**Pull request by shweta83**
Added BZ# 2004158 [https://bugzilla.redhat.com/show_bug.cgi?id=2004158 
](url)

### CODE REVIEW

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_fact.py -k test_positive_custom_facts_and_clean_orphaned_facts

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7668
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_fact.py -k test_positive_custom_facts_and_clean_orphaned_facts --external-logging
Test Result : =========== 1 passed, 9 deselected, 18 warnings in 688.06s (0:11:28) ===========
```

File: tests/foreman/cli/test_fact.py [hunk 1]
```diff
@@ -121,16 +121,22 @@ def test_positive_update_client_facts_verify_imported_values(
...
+    :customerscenario: true
+
     :steps:
         1. Create few custom facts
         2. Verify on Satellite
         3. Run "foreman-rake facts:clean" to clean orphaned facts.
 
-    :expectedresults: Custom facts are created, uploaded successfully and orphaned facts are cleaned up.
+    :expectedresults: Custom facts are created, uploaded successfully and orphaned facts are cleaned up and without
+     any foreign key violation .
```
Comment (vijaysawant) [inline tests/foreman/cli/test_fact.py:139]:
```suggestion
    :expectedresults: Custom facts are created, uploaded successfully and orphaned facts are cleaned up without
     any foreign key violation.
```

**Review by vijaysawant — CHANGES_REQUESTED** [review]
minior changes
Ack

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_fact.py -k test_positive_custom_facts_and_clean_orphaned_facts

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7680
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_fact.py -k test_positive_custom_facts_and_clean_orphaned_facts --external-logging
Test Result : =========== 1 passed, 9 deselected, 18 warnings in 734.05s (0:12:14) ===========
```

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**